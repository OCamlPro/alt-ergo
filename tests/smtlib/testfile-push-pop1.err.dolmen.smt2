(set-logic ALL)

; Pushing an empty assertion stack
(push 1)
(check-sat)

(declare-const x Int)
; Pushing twice the assertion stack with 'x'
(push 2)

; Pushing once the assertion stack with 'y'
(declare-const y Int)
(push 1)

(assert (= x y))
(check-sat)

; Popping three times -> y disappears
(pop 3)

(assert (= x y))
; Raises an error