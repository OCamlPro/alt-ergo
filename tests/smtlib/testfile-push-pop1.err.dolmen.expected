
unknown

unknown
(error "Error on typing errors (code 4)")
